README.md

```diff
@@ -115,37 +116,45 @@ This allows you to start a new giveaway. Once the `start()` function is called,
 </a>
 
 #### ⚠ ATTENTION!
-The command examples below (reroll, edit delete, end) can be executed on any server your bot is a member of if a person has the `prize` or the `messageId` of a giveaway. To prevent abuse we recommend to check if the `prize` or the `messageId` that was provided  by the command user is for a giveaway on the same server, if it is not, then cancel the command execution.
+
+The command examples below (reroll, edit delete, end) can be executed on any server your bot is a member of if a person has the `prize` or the `messageId` of a giveaway. To prevent abuse we recommend to check if the `prize` or the `messageId` that was provided by the command user is for a giveaway on the same server, if it is not, then cancel the command execution.
 
 ```js
-const giveaway = 
-// Search with giveaway prize
-client.giveawaysManager.giveaways.find((g) => g.guildId === interaction.guildId && g.prize === interaction.options.getString('query')) ||
-// Search with messageId
-client.giveawaysManager.giveaways.find((g) => g.guildId === interaction.guildId && g.messageId === interaction.options.getString('query'));
+const giveaway =
+    // Search with giveaway prize
+    client.giveawaysManager.giveaways.find(
+        (g) => g.guildId === interaction.guildId && g.prize === interaction.options.getString('query')
+    ) ||
+    // Search with messageId
+    client.giveawaysManager.giveaways.find(
+        (g) => g.guildId === interaction.guildId && g.messageId === interaction.options.getString('query')
+    );
 
 // If no giveaway was found
-if (!giveaway) return interaction.channel.send('Unable to find a giveaway for `'+ args.join(' ') +'`.');
+if (!giveaway) return interaction.channel.send('Unable to find a giveaway for `' + args.join(' ') + '`.');
```

Comment:
```suggestion
if (!giveaway) return interaction.reply('Unable to find a giveaway for `' + options.getString('query') + '`.');
```
In interaction, it's possible that the bot doesn't have permission to send messages in this channel. So changing it to reply will solve the issue.

Reply:
yeah you can